saleor/dashboard/product/forms.py

```diff
@@ -527,7 +527,7 @@ def save(self):
 class AttributeForm(forms.ModelForm):
     class Meta:
         model = Attribute
-        exclude = ["input_type"]
+        exclude = ["input_type", "product_types", "product_variant_types"]
```

Comment:
`exclude` doesn't make sense here as we only want to allow to change two fields. Instead we should only select the two fields we actually use and expect:

```suggestion
        fields = ["name", "slug"]
```

Reply:
Agreed if those are the only two fields you need. In our project we are also including the new value_required, visible_in_storefront, filterable_in_storefront fields in our Dashboard. If we also want to expose these to Dash 1.0 users I'm happy to put together a simple PR for it.

Reply:
I agree with @NyanKiyoshi. For now, I would go with allowing only those two fields.